src/main/java/appeng/parts/reporting/ConversionMonitorPart.java

```diff
@@ -217,6 +239,23 @@ private void extractItem(Player player, int count) {
         });
     }
 
+    @Override
+    protected void configureWatchers() {
+        super.configureWatchers();
+
+        if (craftingWatcher != null) {
+            craftingWatcher.reset();
+        }
+
+        if (getDisplayed() != null) {
+            if (craftingWatcher != null) {
+                craftingWatcher.add(getDisplayed());
+            }
+
+            getMainNode().ifPresent(this::updateReportingValue);
```

Comment:
This is already done in the super call. You can move the super call to the end. And add a comment saying why we update the crafting watcher first.